dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -654,6 +663,9 @@ private String makeNodeObject()
         sb.append("\",\n \"filePart\" : \"");
         sb.append(this.dataPart.getThisPart());
         sb.append("\", \n");
+        sb.append("\"filePartCopy\" : \"");
```

Comment:
We only want to specify filePartCopy if we are wanting the copy and not the primary correct?